### PR TITLE
ROC-6008 - Fix duplication of claim fee on CCJ PDF for part admission

### PR DESCRIPTION
**Pull request by ellie-harrison**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-6008

### Change description ###
Update CCJ PDF generation for part admission to stop issue where claim fee would be applied to the claim twice

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java [hunk 1]
```diff
@@ -26,6 +26,22 @@ public AmountContentProvider(InterestContentProvider interestContentProvider) {
         this.interestContentProvider = interestContentProvider;
     }
 
+    private BigDecimal getPartAdmissionAmount(Claim claim, PartAdmissionResponse response) {
+        return response.getAmount().subtract(
+            claim.getClaimData().getFeesPaidInPound())
+            .max(new BigDecimal(0));
```
Comment (andrewwa-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java]: Already have the `ZERO` constant

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java [hunk 2]
```diff
@@ -26,6 +26,22 @@ public AmountContentProvider(InterestContentProvider interestContentProvider) {
...
+    }
+
+    private BigDecimal getPartAdmissionClaimFeeInPound(Claim claim,
```
Comment (andrewwa-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java]: Can this be renamed to `...InPounds` please

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java [hunk 3]
```diff
@@ -26,6 +26,22 @@ public AmountContentProvider(InterestContentProvider interestContentProvider) {
         this.interestContentProvider = interestContentProvider;
     }
 
+    private BigDecimal getPartAdmissionAmount(Claim claim, PartAdmissionResponse response) {
+        return response.getAmount().subtract(
+            claim.getClaimData().getFeesPaidInPound())
```
Comment (markry-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java]: doesn't look related to your PR but can we rename `getFeesPaidInPound()` too with `...Pounds` - and is it plural fees?

Comment (klawani) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java]: I can confirm that the requested change has been done properly
